libs/application/templates/new-primary-school/src/lib/newPrimarySchoolUtils.ts

```diff
@@ -127,6 +127,21 @@ export const getApplicationAnswers = (answers: Application['answers']) => {
     'support.hasHadSupport',
   ) as YesOrNo
 
+  const hasWelfareContact = getValueViaPath(
+    answers,
+    'support.hasWelfareContact',
+  ) as YesOrNo
```

Comment:
Shouldn't we also make this use generic types?
```suggestion
  const hasWelfareContact = getValueViaPath<YesOrNo>(
    answers,
    'support.hasWelfareContact',
  )
```